build.sbt

```diff
@@ -4,11 +4,11 @@ import uk.gov.hmrc.DefaultBuildSettings.targetJvm
 val appName = "customs-financials-session-cache"
 
 val silencerVersion = "1.7.16"
-val bootstrapVersion = "8.6.0"
-val scala2_13_12 = "2.13.12"
+val bootstrapVersion = "9.0.0"
```

Comment:
Better to update to 9.1.0 as the latest version is 9.1.0 now. If you face issues after version update then stay with 9.0.0